docs/en/latest/plugins/proxy-rewrite.md

```diff
@@ -66,27 +72,23 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f1
 }'
 ```
 
-## Test Plugin
+## Example usage
```

Comment:
Hi, will you also update all other plugins' structures like this?

Reply:
Yes. That is the plan.